modules/yugabytedb/src/main/java/org/testcontainers/containers/strategy/YugabyteDBYSQLWaitStrategy.java

```diff
@@ -27,10 +28,13 @@
 @Slf4j
 public final class YugabyteDBYSQLWaitStrategy extends AbstractWaitStrategy {
 
-    private static final String YSQL_TEST_QUERY = "SELECT 1";
-
     private final WaitStrategyTarget target;
 
+    private static final String YSQL_EXTENDED_PROBE =
+        "CREATE TABLE IF NOT EXISTS YB_SAMPLE(k int, v int, primary key(k, v))";
+
+    private static final String YSQL_EXTENDED_PROBE_DROP_TABLE = "DROP TABLE IF EXISTS YB_SAMPLE";
```

Comment:
Why switching to temporal table from the original PR? I'm a little bit worried about not fixing the issue with those changes.

Reply:
Hey @eddumelendez 
Regarding switching from temp table, as I have mentioned [here](https://github.com/testcontainers/testcontainers-java/pull/7599#issuecomment-1779165537) I have raised a GH issue https://github.com/yugabyte/yugabyte-db/issues/19629 to track this.
Thank you for accepting the PR.